playbooks/provisioning/openstack/README.md

```diff
@@ -0,0 +1,113 @@
+# OpenStack Provisioning
+
+This repository contains playbooks and Heat templates to provision
+OpenStack resources (servers, networking, volumes, security groups,
+etc.). The result is an environment ready for openshift-ansible.
+
+
+## Dependencies
+
+* [Ansible 2.3](https://pypi.python.org/pypi/ansible)
+* [shade](https://pypi.python.org/pypi/shade)
+
+
+## What does it do
+
+* Create Nova servers with floating IP addresses attached
+* Assigns Cinder volumes to the servers
+* Set up an `openshift` user with sudo privileges
+* Optionally attach Red Hat subscriptions
+* Set up a bind-based DNS server
+* When deploying more than one master, set up a HAproxy server
+
+
+## Set up
+
+### Copy the sample inventory
+
+    cp openshift-ansible-contrib/playbooks/provisioning/openstack/sample-inventory inventory
+
+### Copy clouds.yaml
+
+    cp openshift-ansible-contrib/playbooks/provisioning/openstack/sample-inventory/clouds.yaml clouds.yaml
+
+### Update `inventory/group_vars/all.yml`
+
+Pay special attention to the values in the first paragraph -- these
+will depend on your OpenStack environment.
+
+The `env_id` and `openstack_dns_domain` will form the DNS domain all
+your servers will be under. With the default values, this will be
+`openshift.example.com`.
```

Comment:
Let's insist for the guide to update it to something like `openshift.cluster.lc`, which has nothing to publicly DNS-resolvable domains.